### PR TITLE
updated reminder.txt

### PR DESCRIPTION
**Pull request by user-64bit**

### CODE REVIEW

**Review by ykdojo — APPROVED** [review]

**Comment by user-64bit** [comment]
Hey, I've no idea what to do next?
so please guide me

File: reminders.txt [hunk 1]
```diff
@@ -25,4 +25,4 @@ Hey human, remember to tell your heart that ALL IS WELL 💚.
 Hey human, pursue for excellence and success wil chase you ✨.
 Hey human, remember to eat some fruits and stay healthy ok
 Hey human, if you feel stressed, it's ok to reach out to your loved ones :)
-Hey human, remember to be patient, good things take time.
+Hey human, just remember - no excuses! Dw, you got this 💪
```
Comment (Pradumnasaraf) [inline reminders.txt:29]:
```suggestion
Hey human, remember to be patient, good things take time.
Hey human, just remember - no excuses! Dw, you got this 💪
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Please don't change the already committed line, add in a new line

- You can click on commit suggestions to fix that

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by user-64bit** [comment]
Now? 
Why my quote hasn't been added to the reminder.txt yet?
I'm noob so guide me please

**Comment by Pradumnasaraf** [comment]
> Now? Why my quote hasn't been added to the reminder.txt yet? I'm noob so guide me please

As soon as the PR will get merged into the main branch, it will start show up.

**Comment by user-64bit** [comment]
> > Now? Why my quote hasn't been added to the reminder.txt yet? I'm noob so guide me please
> 
> As soon as the PR will get merged into the main branch, it will start show up.

Thanks, Buddy for your help

**Pull request merged**